bench/Libraries/Microsoft.AspNetCore.Diagnostics.Middleware.PerformanceTests/Microsoft.AspNetCore.Diagnostics.Middleware.PerformanceTests.csproj

```diff
@@ -6,6 +6,7 @@
 
   <PropertyGroup>
     <InjectSharedPools>true</InjectSharedPools>
+    <SkipSharedPoolsReferences>true</SkipSharedPoolsReferences>
```

Comment:
@xakep139 could you please elp me understand the connection between this and the rest of the change?

Reply:
Sure thing. There was a warning complaining about an unused package in this project. I added a condition into the `Shared.props` to avoid touching other projects.

Reply:
You can see the warning here: https://github.com/dotnet/extensions/runs/25307769976

Reply:
Thank you